data_steward/spec/pages/file_transfer_procedures.md

```diff
@@ -1,43 +1,98 @@
-     * The name of your site's assigned bucket on Google Cloud Storage
-     * A test data set to be used during alpha
-     * Information on how to activate your account on our [JIRA site](https://precisionmedicineinitiative.atlassian.net) where you can report and track issues related to this process
-  
-      Keep this information in a safe place and do not share it with others.
+---
+## PROVISIONING
 
-## Upload the Test Data Set
+__Step 1:__ Request creation of pmi-ops.org account for Data Steward(s) by sending an email to sysadmin@pmi-ops.org with the subject   line “NEW USER REQUEST: Data Steward”. 
 
-__Note: You should not upload real participant data or de-identified data during alpha. You will be given a test data set to use instead.__
+   The email should include the following:
+   * Name and Contact (including institutional email)
+   * Organization, Role, and Justification for Access
+   * Signed copy of DRC IT Rules of Behavior for Privileged Use. Note that the entire document must be attached, not just the signature page.
+   * Attestation of completion of institutional general security training
 
-The instructions below assume the bucket assigned to you was `test-site-bucket-aou-data-steward-nyc`. You should adapted according to the information that was sent to you by `sysadmin@pmi-ops.org`:
+If the bucket will have a Primary as well as a Secondary Data Steward, this should be noted in the new user request email. Requests should be sent separately; bundled new user access requests cannot be processed
 
+__Step 2:__ Request creation of a Google Cloud Storage Bucket by sending an email to sysadmin@pmi-ops.org with the subject line “Bucket Creation Request”. The email should include the following:
 
-1. Open a web browser and navigate to the Google Cloud Storage Browser at `https://console.cloud.google.com/storage/browser/<BUCKET_NAME>` but replace `<BUCKET_NAME>` with the bucket name assigned to your site. For example, if your site was assigned `test-site-bucket-aou-data-steward-nyc`, then it would be [https://console.cloud.google.com/storage/browser/test-site-bucket-aou-data-steward-nyc](https://console.cloud.google.com/storage/browser/test-site-bucket-aou-data-steward-nyc).
-1. Enter your `pmi-ops.org` credentials if prompted.
-1. Create a folder inside the bucket to load the data into.
-Suggested folder naming scheme would be the date of the upload.
-1. In your desktop environment, navigate to the local folder where you've downloaded the test data set.
-1. Drag and drop the test data set files from your local folder into the browser window where the Google Cloud Storage Browser is open to the folder created in step 3.
+   * Name of Organization that will perform the upload
+   * Composition of data upload: Will data be uploaded for only one Organization or for multiple Organizations? If for more than one Organization, also include the names of all Organizations whose data will be uploaded.
```

Comment:
@cmcphilamy Data stewards and DRC would benefit from a shared list of organizations to make sure we're talking about the same thing. Does it make sense to link out to https://allofus.nih.gov/about/program-components/health-care-provider-organizations?

Reply:
I am fine with providing a link as long as its mentioned as a reference not something they should check before sending the email.  I would be worried they would look at the list and send the wrong thing.